book/aliases.md

```diff
@@ -66,6 +66,10 @@ displaying all listed files and folders in a grid.
 When replacing commands it is best to "back up" the command first and avoid recursion error.
 :::
 
+::: tip Note
```

Comment:
Can you remove this tip. I'm not sure it's helpful here since the example doesn't show the use of `do` and may just lead to confusion.